pkg/tests/renderdir/renderdir_test.go

```diff
@@ -73,7 +73,7 @@ func TestKotsRenderDir(t *testing.T) {
 			Name:             spec.Name,
 			RenderDirOptions: spec.RenderDirOptions,
 		}
-
+		test.RenderDirOptions.App.SelectedChannelID = "1vusIYZLAVxMG6q760OJmRKj5i5"
```

Comment:
why is this needed if you're already passing it in the render options?

Reply:
Its not - honestly don't even recall when/why i added this - sorry bout that.